src/anemoi/inference/runner.py

```diff
@@ -312,7 +313,10 @@ def get_most_recent_datetime(input_fields):
 
         most_recent_datetime = get_most_recent_datetime(input_fields)
         reference_fields = [f for f in input_fields if f.datetime()["valid_time"] == most_recent_datetime]
-        prognostic_template = reference_fields[self.checkpoint.variable_to_index["lsm"]]
+        if "lsm" in self.checkpoint.variable_to_index:
+            prognostic_template = reference_fields[self.checkpoint.variable_to_index["lsm"]]
+        else:
+            prognostic_template = reference_fields[self.checkpoint.variable_to_index["PS"]]
```

Comment:
I think it would be better to have a list of possible variables that can be used as a template.
The list could be defined as input variable to run, with default list of possibilities, then this could be further generalised in the future.
`prognostic_template_variable_list=["lsm","PS","sp"]`

Reply:
#29 made this change obsolete.